cmd/validator/credentials/set/process.go

```diff
@@ -313,6 +318,31 @@ func (c *command) generateOperationsFromValidatorAndPrivateKey(ctx context.Conte
 	return nil
 }
 
+func (c *command) generateOperationsFromPrivateKey(ctx context.Context) error {
+	withdrawalAccount, err := util.ParseAccount(ctx, c.privateKey, nil, true)
+	if err != nil {
+		return err
+	}
+
+	for _, validatorInfo := range c.chainInfo.Validators {
+		// Skip validators which withdrawal key don't match with supplied withdrawal account public key.
+		pubkey, err := util.BestPublicKey(withdrawalAccount)
+		if err != nil {
+			return err
+		}
+		withdrawalCredentials := ethutil.SHA256(pubkey.Marshal())
+		withdrawalCredentials[0] = byte(0) // BLS_WITHDRAWAL_PREFIX
```

Comment:
All of this can be moved outside the loop as well.

Reply:
Oh my how I did I missed this... done!